watch-package.js

```diff
@@ -8,16 +8,24 @@ var through = require('through2')
 var npm = process.platform === 'win32' ? 'npm.cmd' : 'npm';
 var nodemon = process.platform === 'win32' ? 'nodemon.cmd' : 'nodemon';
 
-module.exports = function watchPackage(pkgDir, exit) {
+var pkgDir = '';
+var stdin = null;
```

Comment:
The Problem is that if i not create these variables in the Global scope i would need to pass them as parameter to the function. So in this case they are in the module scope and not can be refered from outside.